packages/interactivity/docs/2-api-reference.md

```diff
@@ -363,20 +365,90 @@ store( "myPlugin", {
 </details>
 <br/>
 
-The `wp-on` directive is executed each time the associated event is triggered. 
+≈
 
 The callback passed as the reference receives [the event](https://developer.mozilla.org/en-US/docs/Web/API/Event) (`event`), and the returned value by this callback is ignored.
 
-#### `wp-watch` 
+#### `wp-on-window`
+
+It allows to attach global window events like `resize`, `copy`, `focus` and then execute a defined callback when those happen.
+
+[List of supported window events.](https://developer.mozilla.org/en-US/docs/Web/API/Window#events)
+
+> The syntax of this directive is `data-wp-on--[window-event]` (like `data-wp-on-window--resize`
+or `data-wp-on-window--languagechange`).
+
+_Example of `wp-on-window` directive_
+
+```php
+<div data-wp-on-window--resize="callbacks.logWidth"></div>
+```
+
+<details>
+	<summary><em>See store used with the directive above</em></summary>
+
+```js
+store( "myPlugin", {
+  callbacks: {
+   logWidth() {
+			console.log( 'Window width: ', window.innerWidth );
+		},
+  },
+} );
+```
+
+</details>
+<br/>
+
+≈
```

Comment:
Same here.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/57931/commits/c8635c4132b548f97eb4136c39044881c8221867

Reply:
I can still see this one.